goat.sh

```diff
@@ -173,7 +173,15 @@ handle_input() {
     local exit_status=0
 
     # Test existance of the file with shortcuts
-    [ ! -f "$SHORTCUTS_FILE" ] && touch "$SHORTCUTS_FILE"
+    if [ ! -f "$SHORTCUTS_FILE" ];then 
+	touch "$SHORTCUTS_FILE"
+	create down	~/Downloads &>- 
+	create doc 	~/Documents &>-
+	create root 	/	    &>-     
+	create conf 	~/.config/  &>-
+	create proj 	~/Projects  &>-
```

Comment:
I don't believe this should be a default shortcut - it is not a standard directory for projects to me.